var/ramble/repos/builtin/applications/minixyce/application.py

```diff
@@ -0,0 +1,145 @@
+        workload(workload_name, executables=['generate_' + workload_name, 'execute'])
+
+    all_workloads = cir_workloads + rc_workloads
+    workload_variable('t_start', default='0', description='Start time', workloads=all_workloads)
+    workload_variable('t_step', default='1e-8', description='Time step', workloads=all_workloads)
+    workload_variable('t_stop', default='5e-6', description='Stop time', workloads=all_workloads)
+    workload_variable('tol', default='1e-6', description='Tolerance', workloads=all_workloads)
+    workload_variable('k', default='10', description='Tells GMRES how often to restart', workloads=all_workloads)
+
+    workload_variable('num_ladder_stages', default='8',
+                      description='Number of Ladder stages for RC|RLC ladder|ladder2 test generation',
+                      workloads=rc_workloads)
+
+    log_file = '{experiment_run_dir}/{workload_name}_tran_results.prn'
+    processed_output = '{experiment_run_dir}/processed_output.txt'
+
+    result_regex = r'.*\s+(?P<num_iters>[0-9]+)\s+(?P<num_restarts>[0-9]+)'
+
+    floating_point_regex = r'[\+\-]*[0-9]*\.*[0-9]+E*[\+\-]*[0-9]*'
+    close_paren = r')'
+
+    figure_of_merit('Time', log_file=log_file,
+                    fom_regex=r'\s+(?P<time>' + floating_point_regex + close_paren,
+                    group_name='time',
+                    units='s'
+                    )
+
+    figure_of_merit('num_GMRES_iters', log_file=log_file,
+                    fom_regex=result_regex,
+                    group_name='num_iters',
+                    units=''
+                    )
+
+    figure_of_merit('num_GMRES_restarts', log_file=log_file,
+                    fom_regex=result_regex,
+                    group_name='num_restarts',
+                    units=''
+                    )
+
+    state_var_regex = r'\s*(?P<State_Variable>[0-9]+)*:*\s*(?P<name>[0-9A-Za-z]+)\s*=\s*(?P<value>' + floating_point_regex + r')'
+
+    figure_of_merit('Name', log_file=processed_output,
+                    fom_regex=state_var_regex,
+                    group_name='name',
+                    units='', contexts=['state_variable']
+                    )
+
+    figure_of_merit('Value', log_file=processed_output,
+                    fom_regex=state_var_regex,
+                    group_name='value',
+                    units='', contexts=['state_variable']
+                    )
+
+    figure_of_merit_context('state_variable',
+                            regex=state_var_regex,
+                            output_format='{State_Variable}')
+
+    def _make_experiments(self, workspace):
+        super()._make_experiments(workspace)
+
+        input_path = self.expander.expand_var('{experiment_run_dir}/params.txt')
+
+        settings = ['t_start', 't_step', 't_stop', 'tol', 'k']
+
+        with open(input_path, 'w+') as f:
+            for setting in settings:
+                f.write(setting + ' = ' + self.expander.expand_var('{' + setting + '}') + '\n')
```

Comment:
Please change the '{' + ... + '}' syntax tp use this" https://github.com/GoogleCloudPlatform/ramble/blob/develop/lib/ramble/ramble/expander.py#L190